packages/elements/package.json

```diff
@@ -75,7 +75,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@xstate/react": "^4.1.1",
     "client-only": "^0.0.1",
-    "xstate": "^5.13.0"
+    "xstate": "^5.13.2"
```

Comment:
❓ why did we upgrade this dependency? This does not seem related to the PR.

Reply:
Correct, there were a number of things that became super weird after merging the rebasing main with the Next 14 version update.